<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>718</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1243</width>
      <height>658</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Basic features of constructed wetlands</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Scale:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="10" colspan="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Nitrogen</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Water quality:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="17">
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>PPCPs</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Pesticides</string>
          </property>
         </widget>
        </item>
        <item row="3" column="16">
         <widget class="QSpinBox" name="spinBox_14">
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5" colspan="2">
         <widget class="QSpinBox" name="spinBox_25">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="4" column="8" colspan="2">
         <widget class="QSpinBox" name="spinBox_22">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="5" column="10">
         <widget class="QSpinBox" name="spinBox_26">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="3" column="12" colspan="2">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Water level</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9" colspan="5">
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>1.Free water surface flow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.Horizontal subsurface flow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.Vertical subsurface flow</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="14">
         <widget class="QSpinBox" name="spinBox_13">
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="14" colspan="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Phosphorus</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>1.Laboratory</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.Small</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.Large</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QSpinBox" name="spinBox_21">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="1" column="7" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Flow type</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Hydraulic conditions</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="6" colspan="4">
         <widget class="QCheckBox" name="checkBox_1">
          <property name="text">
           <string>Domestic sewer system</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="10" colspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item row="3" column="11">
         <widget class="QSpinBox" name="spinBox_12">
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="5" column="8" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Heavy metals</string>
          </property>
         </widget>
        </item>
        <item row="3" column="15">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>HRT</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Inflow sources:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="13" colspan="4">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Other natural water bodies</string>
          </property>
         </widget>
        </item>
        <item row="2" column="10" colspan="3">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Industary drainage</string>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QSpinBox" name="spinBox_11">
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>DO</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="9">
         <widget class="QComboBox" name="comboBox_1">
          <item>
           <property name="text">
            <string>1.Stormwater management</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.Municipal wastewater treatment</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.Industrial wastewater treatment</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.Agricultural pollutants removal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5.Purification of natural water bodies</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6.Mine drainage treatment</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7.Ecosystem restoration</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8.User-defined</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="3" colspan="3">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Rainfall</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="5">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Inflow loading rate</string>
          </property>
         </widget>
        </item>
        <item row="4" column="12">
         <widget class="QSpinBox" name="spinBox_23">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="4" column="16">
         <widget class="QSpinBox" name="spinBox_24">
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="4" column="7">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>BOD</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8" colspan="3">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Flow fluctuation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Changing hydrological conditions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_4">
           <item>
            <property name="text">
             <string>1.Wet seasons</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.Dry seasons</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3.Prolonged flooding</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4.Rainstrom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5.Highly fluctuating flow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6.Short-term inundation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7.Circular-flow corridor and baffled subsurface flow CWs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8.Tidal-operated CWs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9.Clogging of porous media</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10.Artificial pulsing in subsurface CWs</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Evaluate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Possible change in hydraulic conditions</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Inflow loading rate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label11">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Flow fluctuation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label12">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Water level</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label13">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>HRT</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label14">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Possible change in water quality</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>DO</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label21">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_37">
              <property name="text">
               <string>BOD</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label22">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_38">
              <property name="text">
               <string>Nitrogen</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label23">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>Phosphorus</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label24">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_39">
              <property name="text">
               <string>Pesticides</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label25">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>Heavy metals</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label26">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
